.github/workflows/build.yml

```diff
@@ -104,13 +104,16 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Configure
-        run: cmake -DPCRE2_SUPPORT_JIT=ON -DPCRE2_BUILD_PCRE2_16=ON -DPCRE2_BUILD_PCRE2_32=ON -DCMAKE_IGNORE_PREFIX_PATH=C:/Strawberry/c -B build -A Win32
+        run: cmake -G "Visual Studio 17 2022" -DBUILD_SHARED_LIBS=ON -DINSTALL_MSVC_PDB=ON -DPCRE2_SUPPORT_JIT=ON -DPCRE2_BUILD_PCRE2_16=ON -DPCRE2_BUILD_PCRE2_32=ON -DCMAKE_IGNORE_PREFIX_PATH=C:/Strawberry/c -B build -A Win32
 
       - name: Build
-        run: cmake --build build
+        run: cmake --build build --config=RelWithDebInfo
+
+      - name: Install
+        run: cmake --install build --config=RelWithDebInfo --prefix=install
 
       - name: Test
         run: |
-          cd build\Debug
+          cd install\bin
```

Comment:
"bin" might change based on other cmake variables, so it might be better to make that setting explicit at configure time to avoid fragility